Library/Dependencies/napi/source/js_native_api_chakra.cc

```diff
@@ -3662,3 +2429,15 @@ napi_status napi_add_finalizer(napi_env env,
                                napi_ref* result) {
   return napi_ok;
 }
+
+napi_status napi_adjust_external_memory(napi_env env,
+                                        int64_t change_in_bytes,
+                                        int64_t* adjusted_value) {
+  CHECK_ARG(env, adjusted_value);
+
+  // TODO(jackhorton): Determine if Chakra needs or is able to do anything here
```

Comment:
Does this need an issue? Also, who's jackhorton?

Reply:
This is from the original source.